.github/workflows/ci.yml

```diff
@@ -101,8 +101,11 @@ jobs:
         with:
           version: latest
           working-directory: ${{ matrix.dir }}
+          # Temporarily disable staticcheck till `elliptic.GenerateKey` is updated to a proper method.
```

Comment:
We should fix the issue or disable the check for that line rather than disable staticcheck altogether. It's okay if you need to submit with that failing here. It looks like we can't do a simple migration since P224 is not supported by crypto/ecdh. So just add the comment to import.go:134
```	//nolint:staticcheck crypto/ecdh does not support P-224, while we do.
	// We need to remove support for curve 224 to remove this comment.
```.

Same with line 137

Reply:
Done. Did some experiments on its format, it turned out 
```
//nolint:staticcheck
// crypto/ecdh does not support P-224, while we do. We need to remove support for curve 224 to remove this comment.
```
is the correct one to skip staticcheck.